content/en/docs/quickstart.md

```diff
@@ -43,86 +43,96 @@ notation list --plain-http $IMAGE
+Use `notation cert generate-test` to generate a test RSA key for signing artifacts, and a self-signed test certificate for verifying artifacts.
 
 **IMPORTANT**: Self-signed certificates should be used for development purposes only and should not be used in production environments.
 
+The following command generates a test key and a self-signed X.509 certificate under the `$HOME/.config/notation/localkeys` directory. The test key is set as a default signing key using `--default` flag. The self-signed certificate is added to a named trust store `wabbit-networks.io` of type `ca`.
+
 ```console
 notation cert generate-test --default "wabbit-networks.io"
 ```
 
-The output of the above command shows the location of the public and private key generated. For example:
+Use `notation key list` to confirm the signing key. Key name with a `*` prefix is the default key.
 
-```output
-$ notation cert generate-test --default "wabbit-networks.io"
-generating RSA Key with 2048 bits
-generated certificates expiring on 2022-10-01T20:18:37Z
-wrote key: <EXAMPLE_PATH>/notation/localkeys/wabbit-networks.io.key
-wrote certificate: <EXAMPLE_PATH>/notation/localkeys/wabbit-networks.io.crt
-wabbit-networks.io: added to the key list
-wabbit-networks.io: marked as default
+```console
+notation key list
 ```
 
-## Sign the image
+Use `notation certificate list` to confirm whether the certificate is stored in the trust store.
+
+```console
+notation cert list
+```
 
-Use `notation sign` to sign the container image.
+## Sign the container image
+
+Use `notation sign` to sign the container image with COSE signature format.
 
 ```console
-notation sign --plain-http $IMAGE
+notation sign --plain-http --signature-format cose $IMAGE
 ```
 
-Use `notation list` to show the signatures for your container image.
+Use `notation list` to show the signatures associated with the container image.
 
 ```console
 notation list --plain-http $IMAGE
 ```
 
-Confirm there is one listed, for example:
+Confirm there is one signature, for example:
 
 ```output
 $ notation list --plain-http $IMAGE
-sha256:1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef
+localhost:5000/net-monitor@sha256:073b75987e95b89f187a89809f08a32033972bb63cda279db8a9ca16b7ff555a
+└── application/vnd.cncf.notary.v2.signature
+    └── sha256:ba3a68a28648ba18c51a479145fca60d96b43dc96c6ab22f412c89ac56a9038b
 ```
 
-## Verify the signature of an image
-
-Use `notification verify` to verify any signatures on your container image.
-
-```console
-notation verify --plain-http $IMAGE
+## Create a trust policy
+
+In order to verify the container image, you need to configure the trust policy to specify trusted identities which sign the artifacts, and level of signature verification to use. See [trust policy spec](https://github.com/notaryproject/notaryproject/blob/main/trust-store-trust-policy-specification.md#trust-policy) to understand more about trust policy.
```

Comment:
There is a [Trust Policy doc](https://notaryproject.dev/docs/concepts/trust-store-trust-policy-specification/#trust-policy) on the Notary website already, we should use a relative link here. You can replace it with `[trust policy spec]({{< ref "/docs/concepts/trust-store-trust-policy-specification/#trust-policy" >}})`.